front/lib/iam/session.ts

```diff
@@ -151,6 +152,7 @@ export function makeGetServerSidePropsRequirementsWrapper<
 >({
   enableLogging = true,
   requireUserPrivilege,
+  requireCanUseProduct = false,
```

Comment:
`nit` We could enforce that this one can only be true if the `requireUserPrivilege` is equal to `user`.